src/main/java/de/tum/in/www1/artemis/service/compass/assessment/Assessment.java

```diff
@@ -56,23 +59,36 @@ public void addFeedback(Feedback feedback, Context context) {
         return this.contextFeedbackList;
     }
 
+    /**
+     * Calculates a score for a given list of feedback elements. The score contains points, a collection of feedback comments and the confidence. Points: the credits that the
+     * maximum amount of feedback elements share. Feedback comments: the collected feedback texts of all feedback elements that share the credits from above. Confidence: the
+     * maximum percentage of feedback elements that share the same credits.
+     *
+     * @param feedbacks the list of feedback elements
```

Comment:
this is a really bad description, because it's completely obvious that we have a list of feedback elements here ;-)
Please improve

Reply:
done